src/Model/DocsAsHtml.hs

```diff
@@ -241,15 +240,6 @@ splitOnPathSep str
   | '\\' `elem` str = splitOn "\\" str
   | otherwise       = [str]
 
-renderFixity :: P.Fixity -> Html ()
-renderFixity (P.Fixity associativity precedence) =
```

Comment:
Yes, I think we're able to. But what point is there in rendering it as another paragraph when the same information is in the declaration already?